runtime/ftplugin/jq.vim

```diff
@@ -0,0 +1,21 @@
+" Vim compiler file
+" Language:	jq
+" Maintainer:	Vito <vito.blog@gmail.com>
+" Last Change:	2024 Apr 17
+" Upstream: https://github.com/vito-c/jq.vim
+
+if exists('b:did_ftplugin')
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:save_cpoptions = &cpoptions
+set cpoptions&vim
```

Comment:
You are not using line-continuation here, so you can remove the save and restore of 'cpoptions'